ipatests/prci_definitions/nightly_ipa-4-8_latest.yaml

```diff
@@ -1341,6 +1341,18 @@ jobs:
         timeout: 4800
```

Comment:
The test_suite should be updated in order to run only the tests without AD trust:
```
        test_suite: test_integration/test_ipahealthcheck.py::TestIpaHealthCheck test_integration/test_ipahealthcheck.py::TestIpaHealthCheckWithoutDNS
```